docker/Dockerfile.pretrained_embeddings_spacy_de

```diff
@@ -23,9 +23,8 @@ RUN . /opt/venv/bin/activate && poetry build -f wheel -n && \
 # make sure we use the virtualenv
 ENV PATH="/opt/venv/bin:$PATH"
 
-# spacy link
-RUN python -m spacy download de_core_news_sm && \
-    python -m spacy link de_core_news_sm de
+# spacy link is deprecated in v3.0
```

Comment:
Does this comment make sense out of the context of this PR?

Reply:
Fair! I'll drop the comment.